docs/integrations/azure_pipelines.md

```diff
@@ -311,6 +311,16 @@ view the [script options output][script_options] for the latest release.
       # Specify multiple explicit dependency file paths.
       - script: phylum-ci --depfile requirements-prod.txt Cargo.toml path/to/dependency.file
 
+      # Exclude dependency files by gitignore-style pattern.
+      - script: phylum-ci --exclude "requirements-*.txt"
+
+      # Specify multiple exclusion patterns.
+      - script: phylum-ci --exclude "build.gradle" "tests/fixtures/*"
```

Comment:
The `test/fixtures/*` pattern seems to be a common example. But does the `*` on the end actually do any work? That pattern is probably better written as `test/fixtures/`, so that's likely what our examples should use.

Reply:
The change has been made.